index.html

```diff
@@ -63,173 +64,168 @@ <h1>AMBROSE ISIGBA</h1>
         <div><img src="./images/image-geometry1.png" alt="" /></div>
       </div>
     </main>
-    <section id="project-sections">
-      <div id="project-container">
-        <h2 id="proj1">Projects</h2>
-        <div class="full-section-div">
-          <div id="img-div1" class="imj-project-section">
-            <img id="desktop-img1a" src="./images/image_dots.png" alt="" />
-            <img id="full-section-div1" src="./images/desktop7.png" alt="" />
-          </div>
-          <div id="list-of-project1" class="list-of-project">
-            <h4>Project names goes here</h4>
-            <ul>
-              <li class="project-list-1">HTML/CSS</li>
-              <li class="project-list-2">Ruby on Rails</li>
-              <li class="project-list-3">Javascricpt</li>
-              <li class="project-list-4">+1</li>
-            </ul>
-            <div class="project-link">
-              <button id="btn-hover" type="button">
-                See this project<img
-                  class="img-list"
-                  src="./images/arrow-right.png"
-                  alt="arrow"
-                />
-              </button>
-            </div>
-          </div>
-        </div>
-        <div class="full-section-div fix">
-          <div id="img-div2" class="imj-project-section">
-            <img id="full-section-div2" src="./images/desktop2.png" alt="" />
-            <img id="list-of-project2-img" src="./images/desktop5.png" alt="" />
-            <img
-              id="list-of-project2-imgb"
-              src="./images/Group-83-copy.png"
-              alt=""
+ <section id="project-sections">
```

Comment:
Also, this whole section is a mess in desktop view. Have a look at the below screenshots. Kindly fix it and check it thoroughly before asking for another review.
![image](https://user-images.githubusercontent.com/33268927/175326739-9a0866f1-7a4d-43dd-99f2-272dbfb3f5b8.png)
![image](https://user-images.githubusercontent.com/33268927/175326819-9949cb95-93b7-4287-b06e-3cfc7542824e.png)